core/src/main/java/com/exadel/aem/toolkit/core/lists/utils/ListResourceUtils.java

```diff
@@ -0,0 +1,121 @@
+package com.exadel.aem.toolkit.core.lists.utils;
```

Comment:
According contributing rules all new files should contain header Apache Licence. Could you please add it?